### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by xendk**
Add a more general first page, tweek header color and add a docker compose file.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+services:
+  mkdocs:
+    image: python:3-alpine
+    volumes:
+      - ./:/docs
+    stdin_open: true
+    tty: true
+    working_dir: /docs
+    # This could be more efficient, but it works.
+    command: 'sh -c "apk add bash git && pip install --no-cache-dir -r requirements.txt && mkdocs serve -a 0.0.0.0:80"'
```
Comment (kasperg) [inline docker-compose.yml:10]: I wonder if you considered using [a prebuilt mkdocs image](https://hub.docker.com/search?q=mkdocs).

There does not seem to be an official one, but building on existing work might be helpful.

File: docs/README.md [hunk 2]
```diff
@@ -1,19 +1,23 @@
 # About
 
-This is the main documentation site for DPL repositories (Danish
-Public Libraries). It contains general DPL and cross repository
-documentation and consolidates documentation from individual
-repositories.
+This is the main documentation site for [Danish Public
+Libraries](https://detdigitalefolkebibliotek.dk/) projects. These
```
Comment (kasperg) [inline docs/README.md:4]: Please consider readding `(DPL)` somewhere to make the relationship between the organisation and the abbreviation clear.

**Review by kasperg — APPROVED** [review]
:+1: LGTM

File: docker-compose.yml (hunk 1 above)
Comment (xendk) [reply docker-compose.yml:10]: The first I tried died on missing the `mkdocs-multirepo-plugin`. The next I looked at talked so much about "syncing your documentation" that I got cold feet. By then I'd wasted so much time that I needed to get moving.

It's worth revisiting at some time, but currently I'm apparently the only one that felt the need, so... :smiley:

File: docs/README.md (hunk 2 above)
Comment (xendk) [reply docs/README.md:4]: Will do.

**Pull request merged**